src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,773 @@
+    				}
+    				$html .= '<a href="'.DataUtil::formatForDisplay($menuitem['url']).'"'.$attr.' style="display: inline-block;">'.$icon.$menuitem['text'].'</a>';
+    				if (isset($menuitem['links'])) {
+    					$html .= '<a href="#" class="dropdown-toggle" data-toggle="dropdown" style="text-decoration: none;">&nbsp;<b class="caret"></b></a>';
+    				}
+    			} else {
+    				$html .= '<span'.$attr.'>'.$menuitem['text'].'</span>';
+    			}
+    			if (isset($menuitem['links'])) {
+    				$html .= '<ul class="dropdown-menu">';
+    				foreach($menuitem['links'] as $submenuitem) {
+    					$html .= '<li>';
+    					if (isset($submenuitem['url'])) {
+    						$html .= '<a href="'.DataUtil::formatForDisplay($submenuitem['url']).'">'.$submenuitem['text'].'</a>';
+    					} else {
+    						$html .= $submenuitem['text'];
+    					}
+    					$html .= '</li>';
+    				}
+    				$html .= '</ul>';
+    			}
+    			$html .= '</li>';
+    		}
+    	
+    		$html .= '</ul>';
+    	}    	
+    	
+    	return ['menu' => $html];
+    }    
+    
+    
+    /**
+     * Get security analyzer data.
+     *
+     * @return array data
+     */
+    private function _securityanalyzer()
+    {
+        $data = array();
+
+        // check for magic_quotes
+        $data['magic_quotes_gpc'] = DataUtil::getBooleanIniValue('magic_quotes_gpc');
+
+        // check for register_globals
+        $data['register_globals'] = DataUtil::getBooleanIniValue('register_globals');
+
+        // check for config.php being writable
+        $data['config_php'] = (bool)is_writable('config/config.php');
+
+        // check for .htaccess in temp directory
+        $app_htaccess = false;
+        $appDir = $this->get('kernel')->getRootDir();
+        if ($appDir) {
+            // check if we have an absolute path which is possibly not within the document root
+            $docRoot = System::serverGetVar('DOCUMENT_ROOT');
+            if (StringUtil::left($appDir, 1) == '/' && (strpos($appDir, $docRoot) === false)) {
+                // temp dir is outside the webroot, no .htaccess file needed
+                $app_htaccess = true;
+            } else {
+                if (strpos($appDir, $docRoot) === false) {
+                    $ldir = dirname(__FILE__);
+                    $p = strpos($ldir, DIRECTORY_SEPARATOR.'system'); // we are in system/Admin
+                    $b = substr($ldir,0 , $p);
+                    $filePath = $b.'/'.$appDir.'/.htaccess';
+                } else {
+                    $filePath = $appDir.'/.htaccess';
+                }
+                $app_htaccess = (bool) file_exists($filePath);
+            }
+        } else {
+            // already customized, admin should know about what he's doing...
+            $app_htaccess = true;
+        }
+        $data['app_htaccess'] = $app_htaccess;
+
+        $data['scactive']  = (bool)ModUtil::available('ZikulaSecurityCenterModule');
```

Comment:
I haven't looked at the `available` method, but doing something like `$this->get('kernel')->getModule('ZikulaSecurityCenterModule');` will tell you if the module is loaded. which may be sufficient.